.github/workflows/issue-cleanup-orchestrator.yml

```
@@ -0,0 +1,52 @@
+        pip install -r requirements.txt || true
+    
+    - name: Run Issue Cleanup
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        echo "🧹 Running automated issue cleanup..."
+        python scripts/automated_issue_cleanup.py
+    
+    - name: Generate Cleanup Report
+      if: always()
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)